Dockerfile

```diff
@@ -8,6 +8,12 @@ EXPOSE 8707
 
 RUN apk add --no-cache curl
 
-ADD blackfire blackfire-agent /usr/bin/
+COPY blackfire blackfire-agent /usr/local/bin/
+
+# Don't run as root
+RUN addgroup -S -g 999 blackfire && adduser -S -G blackfire -u 999 blackfire
```

Comment:
This does not work, the group 999 already exists on alpine

Reply:
OK, I've removed it